src/cloud/benchflow/sender/sender.go

```diff
@@ -196,23 +218,43 @@ func consumeFromTopic(t TransformerSetting) {
 
 func submitAnalyser(script string, trialID string, minioKey string) {
 	var args []string
+	args = append(args, "--jars", sparkHome+"/pyspark-cassandra-assembly-0.2.7.jar")
```

Comment:
Use a shared variable to store the version (e.g., `pysparkCassandraVersion`) and refer to it.